.eslintrc.js

```diff
@@ -60,10 +60,13 @@ module.exports = {
 
     '@typescript-eslint/explicit-module-boundary-types': 'off',
     '@typescript-eslint/no-explicit-any': 'off',
+    '@typescript-eslint/no-non-null-assertion': 'off',
+    '@typescript-eslint/no-non-null-asserted-optional-chain': 'off',
 
     // vue
     'vue/no-v-html': 'off',
     'vue/require-default-prop': 'off',
+    'vue/require-explicit-emits': 'off',
```

Comment:
Cannot be used with variable definition emit, so disable it.
https://github.com/element-plus/element-plus/blob/455aa9822ac6e4211aec4d690ed25ff235dcb8e8/packages/components/button/src/button.vue#L40


In fact, we can use typescript to define emit to check.